tests/test_sheet_access.py

```diff
@@ -0,0 +1,18 @@
+import random
+
+from base import SheetBaseTestcase
+
+
+class SheetAccessTest(SheetBaseTestcase):
```

Comment:
Seems you come from OOP background!

Reply:
Every time I make a class, I will ask myself: will I use any of the OOP features? If answer is NO, I will not use use classes.

Reply:
Please do not take it as criticism. Developers have their own view. I would like to hear more from you. That is an exchange.

Reply:
well, i'm not coming from the java world if that's what you mean... ;)

If you have a hard opinion about it i can revert it to pure functions, but especially in unit tests i liked the idea of having a common base class. For grouping as well as for the common setUp and tearDown functions. I saw you were using them quite frequently too...

Reply:
Yes, I am opinionated when it comes to Java. “Thinking in Java” book had a a whole chapter: everything is object, hence Java developers cannot forget it when they write Python code.

Surely if you use OOP features, then a class will make more sense. What I am object to make a class for the sake of OOP.

Reply:
sure.

i added another test to raise the coverage, and make the point of having a class more clear ;)
Don't know why codecov still fails. any idea?